src/Dropdown.php

```diff
@@ -131,31 +132,54 @@ public static function show($itemtype, $options = [])
         $name         = $params['emptylabel'];
         $comment      = "";
 
+        if ($params['multiple']) {
+            $params['display_emptychoice'] = false;
+            $params['values'] = $params['value'] ?? [];
+            $params['comments'] = false;
+            unset($params['value']);
+        }
+
        // Check default value for dropdown : need to be a numeric (or null)
         if (
-            $params['value'] !== null
+            isset($params['value'])
             && ((strlen($params['value']) == 0) || !is_numeric($params['value']) && $params['value'] != 'mygroups')
         ) {
             $params['value'] = 0;
         }
 
-        if (isset($params['toadd'][$params['value']])) {
+        if (!$params['multiple'] && isset($params['toadd'][$params['value']])) {
             $name = $params['toadd'][$params['value']];
         } else if (
-            ($params['value'] > 0)
-                 || (($itemtype == "Entity")
-                     && ($params['value'] >= 0))
+            !$params['multiple']
+            && $params['value'] > 0
+            || ($itemtype == "Entity" && $params['value'] >= 0)
         ) {
             $tmpname = self::getDropdownName($table, $params['value'], 1);
 
             if ($tmpname["name"] != "&nbsp;") {
                 $name    = $tmpname["name"];
                 $comment = $tmpname["comment"];
             }
+        } else if ($params['multiple']) {
+            $names = [];
+
+            foreach ($params['values'] as $value) {
+                $names[] = self::getDropdownName($table, $value);
+            }
         }
 
         if ($params['readonly']) {
-            return self::getDropdownName($table, $params['value']);
+            if ($params['multiple']) {
+                // Multiple values, print the names separated by commas
+                $names = [];
+                foreach ($params['values'] as $value) {
+                    $names[] = self::getDropdownName($table, $value);
+                }
+                return implode(", ", $names);
+            } else {
+                // Single value, print the matching name
+                return self::getDropdownName($table, $params['value']);
```

Comment:
`$name` is already computed, I guess.
```suggestion
                return $name;
```